ospfd/ospf_apiserver.c

```diff
@@ -1427,45 +1561,20 @@ struct ospf_lsa *ospf_apiserver_opaque_lsa_new(struct ospf_area *area,
 
 int ospf_apiserver_is_ready_type9(struct ospf_interface *oi)
 {
-	/* Type 9 opaque LSA can be originated if there is at least one
-	   active opaque-capable neighbor attached to the outgoing
-	   interface. */
-
-	return (ospf_nbr_count_opaque_capable(oi) > 0);
+	/* We can always handle getting opaque's even if we can't flood them */
+	return 1;
```

Comment:
will receiving a `1` here imply the opaque lsa will be transmitted, or only it's been received? just making certain a user won't misinterpret the response ...